nextflow.config

```diff
@@ -40,20 +32,21 @@ singularity {
 
 // Configuring LSF job submission
 executor {
-    name = 'lsf'
-    perJobMemLimit = true
+  $lsf {
+      perJobMemLimit = true 
+  }
 }
 
 process {
     withLabel: normal {
+        executor = 'lsf'
         queue  = "normal"
-        cpus   = 1
+        cpus   = 4
         memory = 2.GB
     }
-    withLabel: normal4core {
-        queue  = "normal"
+    withLabel: local {
+        executor = 'local'
```

Comment:
4 CPUs on a head node doesn't sound like the right thing to do. This is meant to run from an already submitted job as a nextflow-head-job of sorts, right?